packages/core/package.nls.json

```
@@ -265,5 +265,8 @@
     "AWS.amazonq.learnMore": "Learn More About Amazon Q",
     "AWS.amazonq.codewhisperer.title": "Amazon Q",
     "AWS.amazonq.toggleCodeSuggestion": "Toggle Auto-Suggestions",
-    "AWS.amazonq.toggleCodeScan": "Toggle Auto-Scans"
+    "AWS.amazonq.toggleCodeScan": "Toggle Auto-Scans",
+    "AWS.command.dynamoDb.searchTables": "Search Tables",
+    "AWS.command.dynamoDb.copyArn": "Copy ARN",
+    "AWS.command.dynamoDb.refreshExplorer": "Refresh DynamoDb Explorer"
```

nitpick: I think technically this should be:
```suggestion
    "AWS.command.dynamoDb.refreshExplorer": "Refresh DynamoDB Explorer"
```

---

Updated